CircleciScripts/create_xcframeworks.py

```diff
@@ -68,6 +68,14 @@ def map_framework_to_project(framework_list):
 
 # Archive all the frameworks.
 for framework in xcframeworks:
+    xcframework = f"{XCFRAMEWORK_PATH}{framework}.xcframework"
+
+    if os.path.exists(xcframework):
+        log(f"skipping {framework}...")
+        continue
+    
+    log(f"Creating archives for {framework}")
+
```

Comment:
despite this being a small-ish change, it would be better to cut a new PR for this change to keep it isolated. unless i'm incorrect about this being unrelated.

Reply:
I've removed this unrelated change.